zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1482,9 +1482,9 @@ components:
           type: string
         order:
           type: string
+          default: asc
```

Comment:
🔧 Oh, I think you can even go for an enum and validation is done for you: https://swagger.io/docs/specification/data-models/enums/